packaging/linux/gtk-dialog.py

```diff
@@ -43,6 +43,6 @@ def __init__(self, title, text):
     parser.add_argument('--text', type=str, required=False, default='')
     args = parser.parse_args()
     if args.type == 'question':
-        Question(args.title, args.text.replace('\\n', '\n'))
```

Comment:
I assume this had no effect to begin with? Also in the old situation there would be no dobule backslashes to replace?

Reply:
I think you are confusing it with single quoted strings in bash. In Python `"` and `'` behave the same and support escape characters.

Reply:
no, i meant, even in the old situation the message text would not have had two slashes in them. also in the past the newlines were newline characters. (?)

Reply:
I'm not following.
In Python `'\\n'` becomes `\n` and `'\n'` becomes `<newline>`.
In Shell scripts `"\n"` stays `\n` and `printf "\n"` becomes `<newline>`

Reply:
you are right. i made the wrong assumption about the last line.